arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-preparing-node-config.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'How to Configure a Node Using the Orbit SDK'
+sidebar_label: 'Node Config Generation'
+description: 'How to configure a node using the Orbit SDK'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 4
+---
+
```

Comment:
It'd be good to also add here the initial note linking to the `example`.

Reply:
And this is the example for prepare-node-config: https://github.com/OffchainLabs/arbitrum-orbit-sdk/tree/main/examples/prepare-node-config
(See the comment in the token-bridge page for more context)